README.md

```diff
@@ -34,17 +36,43 @@ The MCP Server plugin automatically sets up necessary endpoints and tools upon i
 
 ### Connecting to the MCP Server
 
-MCP clients can connect to the server using:
+MCP clients can connect to the server using either transport option:
 
+#### Streamable Transport (Recommended)
+- Streamable Endpoint: `<jenkins-url>/mcp-server/streamable`
+- Supports message replay and enhanced session management
+- Better error handling and connection recovery
+
+#### Legacy SSE Transport
 - SSE Endpoint: `<jenkins-url>/mcp-server/sse`
 - Message Endpoint: `<jenkins-url>/mcp-server/message`
+- Maintained for backward compatibility
 
 ### Authentication and Credentials
 
 The MCP Server Plugin requires the same credentials as the Jenkins instance it's running on. To authenticate your MCP queries:
 
 1. **Jenkins API Token**: Generate an API token from your Jenkins user account.
-2. **Basic Authentication**: Use the API token in the HTTP Basic Authentication header. Below is an example of VS code settings.xml
+2. **Basic Authentication**: Use the API token in the HTTP Basic Authentication header. Below are examples for both transport types:
+
+#### Streamable Transport Configuration (Recommended)
+```json
+{
+  "mcp": {
+    "servers": {
+      "jenkins": {
+        "type": "http",
+        "url": "https://jenkins-host/mcp-server/streamable",
```

Comment:
We should be using `/mcp` url since that's the standard url followed by other MCP servers at large, too. Thanks!